wifiphisher/common/phishinghttp.py

```diff
@@ -19,13 +20,24 @@ def get(self, url):
         self.redirect("http://10.0.0.1:8080/")
 
 
-class ShowPhishingPageHandler(tornado.web.RequestHandler):
+class CaptivePortalHandler(tornado.web.RequestHandler):
```

Comment:
@sophron sure, but to me that sounds like another word for CaptivePortalHandler, as we can either redirect packets to the internet or to the captive portal.